### PR TITLE
[WIP] Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by moshloop**
This is a very high-level working document designed to solicit comments more than anything else.

### CODE REVIEW

**Comment by scottrigby** [comment]
@moshloop  Thanks for starting this per the meeting conversation. I think this is a good topic to continue resolving during meetings as well, but good to have a place to discuss async.

**Comment by christianh814** [comment]
I think this is a good start! I wonder if it should be explicitly stated that "All submissions must follow the GitOps Principles" (or something to that effect)?

**Comment by chris-short** [comment]
Is this focused on maintainers AND projects or just projects? I look at CONTRIBUTING.md files as a maintainer a lot.

**Comment by christianh814** [comment]
> Is this focused on maintainers AND projects or just projects? I look at CONTRIBUTING.md files as a maintainer a lot.

I read this as just "people" but not necessarily disqualifies  other projects.

**Comment by chris-short** [comment]
Per meeting on 2022-02-02, this is meta so hang on tight...

We discussed the process for working on docs like this. We know from experience when we start working on docs as a PR leads to a lot of confusion. To mitigate confusion, we're suggesting the following (which will also need to land in this doc as well).

1. Create an issue stating the problem, thing trying to be accomplished, etc. and include a link to a HackMD doc that we can comment/edit.
2. Once the doc is in a fairly stable state, it should become a PR (with mention of the issue) with the intention of simple edits. If it becomes a lot of back and forth, go back to the HackMD and bring in updates once sorted out.
3. Once the doc is finalized and there are no more standing issues the PR is eligible to be merged barring any concerns/objections. (we probably need some lightweight criteria for what is a starter vs. non-starter; which I think is lightly touched on in this doc).

With that being said, @moshloop would you mind creating an issue and HackMD accordingly so we can work on the doc more collaboratively?

**Review by williamcaban — COMMENTED** [review]
/LGTM

**Review by christianh814 — APPROVED** [review]
/LGTM

**Comment by christianh814** [comment]
@scottrigby Thoughts?

**Review by roberthstrand — APPROVED** [review]

**Comment by williamcaban** [comment]
It might be good to bring this in the next meeting and complete the discussion.

**Comment by scottrigby** [comment]
Converted this PR to a draft because it is not complete.

> It might be good to bring this in the next meeting and complete the discussion.

@williamcaban yes, good idea. Just added to next week's agenda.

**Review by scottrigby — COMMENTED** [review]
Here are some practical suggestions for getting this to a place where we can merge it. We can always iterate from there.

File: CONTRIBUTING.md
Comment (scottrigby) [inline CONTRIBUTING.md:41]: Added as a simple list above

```suggestion
## Best Practises / White Papers

## Tutorials and Patterns

Additional Acceptance Criteria:

* The requirements in terms of skills, experience and tools are listed up front
* For vendor technology, a link to to a free trial or sandbox should be available 

## Case Studies

Case studies should focus on how GitOps principles were implemented, as well as the specific outcomes and benefits.

## Tool and Technology Comparison


```

**Review by scottrigby — APPROVED** [review]
I think this looks good, though I also have commits on this from my earlier review suggestions. Please if someone else can approve this let's get it in if ready.

**Comment by todaywasawesome** [comment]
Given all the back and forth and this is a pretty simple commit so I'll go ahead and merge it.

**Review by roberthstrand — APPROVED** [review]

**Pull request merged**
